mobly/controllers/android_device.py

```diff
@@ -879,12 +880,22 @@ def load_config(self, config):
       Error: The config is trying to overwrite an existing attribute.
     """
     for k, v in config.items():
+      if k == config_parser.USE_SNIPPET_CLIENT_V2:
```

Comment:
this is way too complicated../
I don't think a complete new channel needs to be added just to pass this one param..
Can we not simplify this to minimize the amount of code changes made in the existing code?

Reply:
Yes, I find that the `BaseTest` can directly set the param to `SnippetManagementService`, adding a new PR #803. 

That PR mainly modifies 2 places. Although it still has 140 lines of addition, but most of them are unit tests and comments.